histogram/src/builder.rs

```diff
@@ -0,0 +1,78 @@
+use crate::Instant;
+use crate::UnixInstant;
+use crate::BuildError;
+
+/// A builder that can be used to construct a sliding window histogram.
+///
+/// By using the `Builder` you can specify a start instant for the histogram.
+pub struct Builder {
```

Comment:
Stupid question: how is this builder different from the one in `sliding_window.rs:28`? My sense is that this entire file is probably leftover from the time of multiple histogram implementations, but I'm not sure.